Automatic Validation ended with:
> 2024-04-23T11:11:15.7523344Z ##[error]Manifest Validation Failed
 2024-04-23T11:11:20.8200337Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/11/Full/11.0.23.10/BellSoft.LibericaJDK.11.Full.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/11/Full/11.0.23.10/BellSoft.LibericaJDK.11.Full.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 11 Full (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-i586-full.msi
-InstallerSha256: AEFFE3CFE3EF3917203D13F36BBC9CE3623CDAF28C3AE418ECB75A55E1231FB2
-ProductCode: '{B751C82B-06F7-428C-B66C-3F58B26984EA}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 11 Full (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-i586-full.msi
+  InstallerSha256: AEFFE3CFE3EF3917203D13F36BBC9CE3623CDAF28C3AE418ECB75A55E1231FB2
+  ProductCode: '{B751C82B-06F7-428C-B66C-3F58B26984EA}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 11 Full (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/11.0.23+10/bellsoft-jdk11.0.23+10-windows-aarch64-full.msi
   InstallerSha256: A1F723F12D52D47D3AFA5A854D8F532B4029779EE55E342F2EFDD2772A65C928

